spec/models/peer_review_spec.rb

```diff
@@ -88,4 +88,23 @@
       end
     end
   end
+
+  describe '#has_marks_or_annotations?' do
```

Comment:
This set of test cases is incomplete, you should be adding more cases (don't just rely on code coverage to tell you whether the tests are complete or not)